src/Avalonia.Base/Input/GestureRecognizers/ScrollGestureRecognizer.cs

```diff
@@ -13,7 +14,8 @@ public class ScrollGestureRecognizer : GestureRecognizer
         private bool _canHorizontallyScroll;
         private bool _canVerticallyScroll;
         private bool _isScrollInertiaEnabled;
-        private int _scrollStartDistance = 30;
+        private readonly static int s_defatultScrollStartDistance = (int)((AvaloniaLocator.Current?.GetService<IPlatformSettings>()?.GetTapSize(PointerType.Touch).Height ?? 10) / 2);
```

Comment:
spelling `s_defatultScrollStartDistance` default is wrong.

Reply:
Fixed